build/modules/casino-modules/COMEON/services/loginService.js

```diff
@@ -0,0 +1,36 @@
+// THIS IS EXAMPLE CODE
+const fetch = require("node-fetch");
+const {
+  getCookieStringFromResponse
+} = require("../../../utilities/cookieHelper");
+const { userAgentHeader, jsonHeader } = require("../../../utilities/headers");
+
+async function login(user) {
+  const body = {
```

Comment:
are captcha & deviceid not required?

Reply:
deviceId can be blank upon login but not on registration, captchaToken can be blank as well it seems.